certified-connectors/OpenLegacy IBM Mainframe/apiDefinition.swagger.json

```diff
@@ -0,0 +1,769 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "OpenLegacy IBM Mainframe",
+        "description": "OpenLegacy IBM Mainframe connector provides bidirectional connectivity to your system’s data and business logic through APIs deployed on OpenLegacy’s cloud or any other target deployment. You can now build Business Flows with information from your legacy environment using the OpenLegacy connector. This will enable you to utilize your legacy assets as digital components, providing you a legacy integration and modernization solution from Day 1.",
+        "version": "1.0",
+        "contact": {
+            "name": "OpenLegacy Support",
+            "url": "https://www.openlegacy.com/company/support",
+            "email": "support@openlegacy.com"
+        }
+    },
+    "host": "api.ol-hub.com",
+    "basePath": "/backend",
+    "schemes": [
+        "https"
+    ],
+    "x-ms-connector-metadata": [
+        {
+            "propertyName": "Website",
+            "propertyValue": "https://www.openlegacy.com"
+          },
+          {
+            "propertyName": "Privacy policy",
+            "propertyValue": "https://www.openlegacy.com/privacy-policy"
+          },
+          {
+            "propertyName": "Categories",
+            "propertyValue": "Business Management; Data"
+          }
+        ],
+    "consumes": [
+        "application/json"
+    ],
+    "produces": [
+        "application/json"
+    ],
+    "paths": {
+        "/api/v1/projects": {
+            "get": {
+                "summary": "Get All Projects",
+                "operationId": "GetAllProjects",
+                "x-ms-visibility": "internal",
+                "parameters": [
+                    {
+                        "name": "with_contracts",
+                        "type": "boolean",
+                        "in": "query",
+                        "default": true,
+                        "x-ms-visibility": "internal",
+                        "x-ms-summary": "Determines that Contract ID must be included in the response.",
+                        "description": "Determines that Contract ID must be included in the response.",
+                        "required": true
+                    },
+                    {
+                        "name": "module_type",
+                        "type": "string",
+                        "in": "query",
+                        "x-ms-visibility": "internal",
+                        "x-ms-summary": "Determines the BackendSolution of module by which projects must be filtered.",
+                        "description": "Determines the BackendSolution of module by which projects must be filtered.",
+                        "required": true
+                    }
+                ],
+                "description": "Returns a list of user's projects",
+                "responses": {
+                    "200": {
+                        "description": "default",
```

Comment:
@boiv may I ask you please change the description from "default" to something like "OK" or "Success". Please fix all responses in the swagger file.

Reply:
@Amjed-Ayoub 
Updated description with value OK in all responses